pkg/models/datasource_cache.go

```diff
@@ -51,7 +51,7 @@ func (ds *DataSource) GetHttpClient() (*http.Client, error) {
 	}
 
 	return &http.Client{
-		Timeout:   30 * time.Second,
+		Timeout:   time.Duration(setting.DataProxyTimeout) * time.Second,
```

Comment:
Im... amazed that we didn't change this before? I know we tried but I guess we updated the code in the wrong place? I'm quite puzzled. Great Work! :)